NEWS.md

```diff
@@ -1,3 +1,7 @@
+# etnservice 0.3.0
+
+- Added `depth_in_meters` field to `get_acoustic_detections()` to achieve feature parity with the etn package (#72, [inbo/etn#274](https://github.com/inbo/etn/pull/274)).
```

Comment:
I updated this, so the multiple references to issues/PRs are included in the same parenthesis (common practice).